start.py

```diff
@@ -1,4 +1,4 @@
 from bot import __main__ as main
 
 if __name__ == "__main__":
-    main()
+    main.main()
```

Comment:
Since we start the bot with `pipenv run start` or directly by `py -m bot` there is no real reason to even keep`start.py`, it can be removed fully.

Reply:
Okay. This file was a pain in the ass